extensions/rich_text_components/Video/Video.html

```diff
@@ -1,7 +1,8 @@
 <script type="text/ng-template" id="richTextComponent/Video">
   <center>
-    <iframe width="500" height="280"
-            ng-src="<[videoUrl]>" frameborder="0" allowfullscreen tabindex="<[tabIndexVal]>">
-    </iframe>
+    <div style="position: relative; padding-bottom: 60.9%;">
```

Comment:
Could you please add a HTML comment here referencing the article -- something along the following lines?

    <!-- The value of 60.9% is derived from the ratio of the height to the width (280/480). See [link] for more details. -->

Then future developers will know where the 60.9% comes from, and can change it if necessary.